pkg/translator/azure/resourcelogs_to_logs.go

```diff
@@ -125,14 +126,27 @@ func (r ResourceLogsUnmarshaler) UnmarshalLogs(buf []byte) (plog.Logs, error) {
 	return l, nil
 }
 
-// asTimestamp will parse an ISO8601 string into an OpenTelemetry
+func getTimestamp(record azureLogRecord) (pcommon.Timestamp, error) {
+	var s string
+	if record.Time != "" {
+		s = record.Time
+
+	} else if record.Timestamp != "" {
+		s = record.Timestamp
+	}
+
+	return asTimestamp(s)
+}
+
+// getTimestamp will parse an ISO8601 string into an OpenTelemetry
```

Comment:
```suggestion
// asTimestamp will parse an ISO8601 string into an OpenTelemetry
```

Reply:
It was renamed, but then I decided to approach the problem differently. Changed back.